app/javascript/react/components/Tabs/Tabs.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const Tabs = () => {
+  return (
+    <div className='container'>
+      <h1>Tabs 1</h1>
```

Comment:
again - time to actually build what we want

remember all placeholder text is causing work for us later on